FoodFriends/app/src/main/java/com/example/foodfriends/fragments/ProfileFragment.java

```diff
@@ -94,11 +97,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
      * Also sets the on click listener for necessary objects
      * Connects the recycler view of restaurants to the adapter
      * Populates the list of restaurants for the adapter to display
-     * */
+     */
     @Override
     public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         currentUser.addObserver(this);
+        loggedInUser = new UserObservable(ParseUser.getCurrentUser());
         rvRestaurants = view.findViewById(R.id.rvProfilePosts);
```

Comment:
Same comment about moving these to onCreateView

Reply:
Done!